location/views.py

```diff
@@ -102,7 +102,7 @@ class LocationDelete(LoginRequiredMixin, BaseDeleteView):
     success_url = reverse_lazy('list-location')
     template_name = 'location/delete.html'
 
-    def get_cancel_link(self):
+    def get_cancel_url(self):
```

Comment:
this function was not referenced anywhere, so I renamed it to bring it in line with the naming convention for Organizations

Reply:
Looking through the rest of the PR, it seems like you successfully adjusted the template, so the only relevant portion of my comment here is related to reintroducing the interface method on the parent class (https://github.com/security-force-monitor/sfm-cms/pull/600#discussion_r315921633).